src/prometheus-mcp-server/awslabs/prometheus_mcp_server/server.py

```diff
@@ -618,14 +674,135 @@ async def get_server_info(ctx: Context) -> ServerInfo:
         raise
 
 
+@mcp.tool(name='GetAvailableWorkspaces')
+async def get_available_workspaces(
+    ctx: Context,
+    region: Optional[str] = Field(None, description='AWS region (defaults to current region)'),
+) -> Dict[str, Any]:
+    """List all available Prometheus workspaces in the specified region.
+
+    ## Usage
+    - Use this tool to see all available Prometheus workspaces
+    - Shows workspace ID, alias, status, and creation date
+    - Use this to find workspace IDs needed for other tools
+    - Always call this first to get a workspace ID before using other tools
+
+    ## Example
```

Comment:
The examples provided in the docs for all the tools do not seem correct. These are supposed to be consumed primarily by the model for invoking the tools, whereas what I see are examples for Python code to invoke it. I would suggest removing these examples.